documentation/homework/week3/todo/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import './App.css';
+import moment from 'moment'
+
+// Components
+import AssignItem from './components/Assign'
+import ListItem from './components/ListItem'
+
+// Params
+import Params from './Params/locals'
+
+class App extends Component {
+  state = {
+    list: []
+  }
+  componentWillMount() {
+    this.setState({
+      list: Params.getLocals || []
+    })
+  }
+  Moments(date) {
```

Comment:
Not totally sure what is going on here, but I think you would solve this problem by simply using a standard date input instead of a text input.